### ISSUE TITLE
Camera missing from Home app, even though it's discovered in plugin

### ISSUE THREAD

**Issue by gergelyszabo94** [hjdhjd/homebridge-unifi-protect#1127]
### Homebridge UniFi Protect Version

v7.5.1

### Homebridge Platform and OS

RPI linux

### Homebridge Version

v1.8.4

### Node Version

v20.17.0 

### UniFi OS Version

4.0.6

### UniFi Protect Controller Version

4.1.53

### Describe the problem

I have 6 cameras, they all show up in the logs, but one of them just won't show in the Home app. I tried removing the bridge and adding it back again, but no luck with that, still only 5 show up. It's the " Frontyard Ret. Wall Corner [G3 Flex]"

### Homebridge HBUP JSON configuration

```json
{
    "name": "UniFi Protect",
    "controllers": [
        {
            "address": "192.168.1.1",
            "username": "homebridge",
            "password": ""
        }
    ],
    "platform": "UniFi Protect",
    "options": [
        "Enable.Device.Standalone.6022321E6159",
        "Enable.Device.SyncName.6022321E6159"
    ],
    "_bridge": {
        "username": "0E:A8:11:A2:C8:8E"
    }
}
```

### Relevant log output

```shell
[27/09/2024, 01:11:04] [UniFi Protect] Initializing UniFi Protect platform...
[27/09/2024, 01:11:04] [UniFi Protect] Initializing child bridge 0E:A8:11:A2:C8:8E
[27/09/2024, 01:11:05] [UniFi Protect] Launched child bridge with PID 913635
[27/09/2024, 01:11:05] [UniFi Protect] Loaded homebridge-unifi-protect v7.5.1 child bridge successfully
[27/09/2024, 01:11:06] [UniFi Protect] Using FFmpeg version: 6.0-homebridge-alpine-aarch64-static.
[27/09/2024, 01:11:06] 

NOTICE TO USERS AND PLUGIN DEVELOPERS
> Homebridge 2.0 is on the way and brings some breaking changes to existing plugins.
> Please visit the following link to learn more about the changes and how to prepare:
> https://github.com/homebridge/homebridge/wiki/Updating-To-Homebridge-v2.0

[27/09/2024, 01:11:06] [UniFi Protect] Dream Machine Special Edition [UDM-PRO-SE]: Connected to 192.168.1.1 (UniFi Protect 4.1.53 running on UniFi OS 4.0.6).
[27/09/2024, 01:11:06] [UniFi Protect] Dream Machine Special Edition [UDM-PRO-SE]: Discovered nvr: Dream Machine Special Edition [UDM SE] (address: 192.168.1.1 mac: 6022321E6159).
[27/09/2024, 01:11:06] [UniFi Protect] Dream Machine Special Edition [UDM-PRO-SE]: Discovered camera: Backyard Driveway Corner [G3 Flex] (address: 192.168.1.36 mac: E438830FDE96).
[27/09/2024, 01:11:06] [UniFi Protect] Dream Machine Special Edition [UDM-PRO-SE]: Discovered camera: Backyard Terrace Stairs Corner [G3 Flex] (address: 192.168.1.247 mac: E438830FD0BA).
[27/09/2024, 01:11:06] [UniFi Protect] Dream Machine Special Edition [UDM-PRO-SE]: Discovered camera: Frontyard Driveway Corner [G3 Flex] (address: 192.168.1.57 mac: E438830FD5B6).
[27/09/2024, 01:11:06] [UniFi Protect] Dream Machine Special Edition [UDM-PRO-SE]: Discovered camera: Frontyard Ret. Wall Corner [G3 Flex] (address: 192.168.1.175 mac: E438830FD105).
[27/09/2024, 01:11:06] [UniFi Protect] Dream Machine Special Edition [UDM-PRO-SE]: Discovered camera: Baby Monitor [G4 Instant] (address: 192.168.1.184 mac: AC8BA90D0A09).
[27/09/2024, 01:11:06] [UniFi Protect] Dream Machine Special Edition [UDM-PRO-SE]: Discovered camera: Bedroom Baby Monitor [G5 Turret Ultra] (address: 192.168.1.149 mac: F4E2C67EAD4A).
[27/09/2024, 01:11:06] [UniFi Protect] Dream Machine Special Edition [UDM-PRO-SE]: Backyard Driveway Corner [G3 Flex]: Adding camera to HomeKit as a standalone device.
[27/09/2024, 01:11:06] [UniFi Protect] Dream Machine Special Edition [UDM-PRO-SE]: Syncing Protect device names to HomeKit.
[27/09/2024, 01:11:06] [UniFi Protect] Dream Machine Special Edition [UDM-PRO-SE]: Backyard Terrace Stairs Corner [G3 Flex]: Adding camera to HomeKit as a standalone device.
[27/09/2024, 01:11:06] [UniFi Protect] Dream Machine Special Edition [UDM-PRO-SE]: Frontyard Driveway Corner [G3 Flex]: Adding camera to HomeKit as a standalone device.
[27/09/2024, 01:11:06] [UniFi Protect] Dream Machine Special Edition [UDM-PRO-SE]: Frontyard Ret. Wall Corner [G3 Flex]: Adding camera to HomeKit as a standalone device.
[27/09/2024, 01:11:06] [UniFi Protect] Dream Machine Special Edition [UDM-PRO-SE]: Baby Monitor [G4 Instant]: Adding camera to HomeKit as a standalone device.
```

### Acknowledgment that you are only running UniFi OS and UniFi Protect releases from the Ubiquiti Official release channel

- [X] I acknowledge that I am not running any early access / beta firmwares or operating systems from either Ubiquiti or Apple.

**Comment by hjdhjd** [comment]
It’s absolutely correct. You decided to unbridge the camera, thereby creating problems for yourself. It’s an advanced option. You’ll also need to deal with the Homebridge settings, using the Homebridge webUI and unpairing/removing whatever bridges you’re dealing with.

TL;DR: you’ve created issues in your environment because you’re trying to unbridge cameras and not appropriately do all the things you need to do to “reset” environments in HomeKit and Homebridge so you can re-pair them. This is not an HBUP support issue…I’ll refer you to poking through the Homebridge documentation and wikis about managing bridges.

Best of luck!

**Comment by gergelyszabo94** [comment]
I think you misunderstood, I started doing things because I had an issue I was trying to resolve. What I meant is that I removed the this plugin's bridge from the Home app and readded it again. Now I removed the bridge again from the home app as well as every camera individually, reinstalled this plugin and readded everything, it's all good now.
